setup.py

```diff
@@ -25,7 +25,7 @@
 
     # You can just specify the packages manually here if your project is
     # simple. Or you can use find_packages().
-    packages=find_packages(exclude=['contrib', 'docs', 'test']),
+    packages=find_packages(exclude=['contrib', 'docs', 'test*']),
```

Comment:
As mentioned in #495 can we instead use:

```suggestion
    packages=find_packages(exclude=['contrib', 'docs', 'test', 'test.*']),
```

Whichever change is made first wins :)